hs_core/templates/pages/my-resources.html

```diff
@@ -34,7 +34,7 @@ <h2 class="page-title">Resources</h2>
                     <th>Title</th>
                     {# <th><i class="text-warning fa fa-star"></i></th> #}
                     <th>Type</th>
-                    <th>Owner</th>
+                    <th>First author</th>
```

Comment:
Author and owner are not the same thing.  Owner pertains to rights within HydroShare.  Author pertains to intellectual contribution over the content and is synonymous with Creator in Dublin core.  These need to be recorded separately.  They may be the same person, but can not be assumed to be the same person.  The request to Hong was an attempt to align the header with what is actually in the column, but now I am not sure that owners and creators are consistently managed and am not clear what was really in the column.